addons/sourcemod/scripting/l4d2_nosey_parker.sp

```diff
@@ -190,4 +196,4 @@ ClearTimer(&Handle:timer)
         KillTimer(timer);
         timer = INVALID_HANDLE;
```

Comment:
I'm somehow surprised that you can pass by reference in sourcepawn... very spooky editing parameters inside functions but :shrug: this makes sense for cleanup